packages/uswds/src/Widgets/CheckboxesWidget.tsx

```diff
@@ -0,0 +1,65 @@
+import { ChangeEvent, FocusEvent } from 'react';
+import { FormContextType, RJSFSchema, StrictRJSFSchema, WidgetProps } from '@rjsf/utils';
+import { Checkbox } from '@trussworks/react-uswds';
+
+export default function CheckboxesWidget<
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any,
+>({
+  id,
+  disabled,
+  options,
+  value = [],
+  readonly,
+  onChange,
+  onBlur,
+  onFocus,
+}: WidgetProps<T, S, F>) {
+  const { enumOptions = [], enumDisabled, inline } = options;
+
+  function _onChange(index: number) {
```

Comment:
There are a set of functions `enumOptionsDeselectValue`,  `enumOptionsIsSelected`,  `enumOptionsSelectValue`,  `enumOptionsValueForIndex` that do a lot of work for you, please use them. See any of the other theme's implementations for how.